jenkins/shared-library/vars/codeCoverage.groovy

```diff
@@ -46,9 +52,10 @@ void verifyCodeCoverageResult(String tool, Integer minimumCodeCoverage) {
 		},
 		'golang': { Integer target ->
 			logCodeCoverageMinimum(target)
-			runScript('go tool cover -func coverage.out')
-			String coverage = runScript("go tool cover -func coverage.out | grep total | awk '{print substr(\$3, 1, length(\$3)-3)}'", true)
-			if (coverage.toInteger() < target) {
+			String coverageOutput = runScript('go tool cover -func coverage.out', true)
+			println(coverageOutput)
```

Comment:
remove ?

Reply:
no, wanted to print the coverage results in the logs